Rakefile

```diff
@@ -0,0 +1,13 @@
+namespace :db do
+  desc "Run migrations"
+  task :migrate, [:version] do |_t, args|
+    require "sequel/core"
+    require 'dotenv/load'
```

Comment:
Use `''` or `""` everywhere, no need to mix styles

Reply:
Fixed, thanks :)